scripts/health-check-core.sh

```
@@ -9,6 +9,7 @@ set -euo pipefail
 RED='\033[0;31m'
 GREEN='\033[0;32m'
 YELLOW='\033[1;33m'
+BLUE='\033[0;34m' # shellcheck disable=SC2034 — used in echo -e strings
```

Shellcheck (0.11.0)

[error] Place shellcheck directives before commands, not after.

(SC1126)